eevee/metrics/asr.py

```diff
@@ -273,7 +273,11 @@ def _preprocess(
         truth = truth_transform(truth)
     else:
         truth = [""]
-    hypothesis = hypothesis_transform(hypothesis)
+
+    if hypothesis.strip() not in [" ", ""]:
```

Comment:
@sachinkum `.strip` will throw error if it's list.

Reply:
Oh yeah. Shall I remove the List option? We're not sending it lists so just string should be fine

Reply:
yes, can remove. can remove from everywhere then.